moneo.py

```diff
@@ -4,97 +4,115 @@
+
+    dep_cmd = dep_cmd + ' -e "skip_insights=' + \
+        ('false' if args.insights else 'true') + '"'
+    dep_cmd = dep_cmd + ' -e "enable_profiling=' + \
+        ('true' if args.profiler_metrics else 'false') + '"'
 
     print('Deployment type: ' + args.type)
     os.system(dep_cmd)
-    
+
+
 def stop(args):
     '''Stops Moneo monitoring on hosts listed in the specified host ini file'''
-    while(True) :
-        confirm = input("Are you sure you would like to perform a '"+ args.type +"' shutdown of Moneo? (Y/n)\n")
-        
-        if confirm.upper() == 'Y' :    
-            dep_cmd='ansible-playbook -i ' + args.host_ini + ' src/ansible/shutdown.yaml'
-            if args.type=='workers':
-                dep_cmd= dep_cmd + ' -e "skip_master=true"'
+    while True:
+        confirm = input("Are you sure you would like to perform a '" + args.type + "' shutdown of Moneo? (Y/n)\n")
+
+        if confirm.upper() == 'Y':
+            dep_cmd = 'ansible-playbook -i ' + args.host_ini + \
+                ' src/ansible/shutdown.yaml'
+            if args.type == 'workers':
+                dep_cmd = dep_cmd + ' -e "skip_master=true"'
             elif args.type == 'manager':
-                dep_cmd= dep_cmd + ' -e "skip_worker=true"'
+                dep_cmd = dep_cmd + ' -e "skip_worker=true"'
             os.system(dep_cmd)
             print("Moneo is Shutting down \n")
             return 0
-        
+
         elif confirm.upper() == 'N':
             print("Canceling request to shutdown Moneo \n")
             return 0
         else:
             print("Input not recognized\n")
 
+
 def jobID_update(args):
     '''Updates job id for hosts listed in the specified host ini file'''
-    dep_cmd='ansible-playbook -i ' + args.host_ini + ' src/ansible/updateJobID.yaml -e job_Id=' + args.job_id
-        
+    dep_cmd = 'ansible-playbook -i ' + args.host_ini + \
+        ' src/ansible/updateJobID.yaml -e job_Id=' + args.job_id
     print('Job ID update to ' + args.job_id)
     os.system(dep_cmd)
 
-def check_deploy_shutdown(args,parser):
-    '''Checks if the necessary arguments are provided for deploy and shutdwon'''
-    if (not os.path.isfile(args.host_ini)) :
+
+def check_deploy_shutdown(args, parser):
+    '''
+    Checks if the necessary arguments are provided for
+    deploy and shutdwon
+    '''
+    if (not os.path.isfile(args.host_ini)):
         print(args.host_ini + " does not exist. Please provide a host file. i.e. host.ini.\n")
         parser.print_help()
         exit(1)
-    if(args.job_id):
+    if args.job_id:
         print("Job Id cannot be specified during deployment and shutdown. Ignoring Job Id.\n")
-    choices=['manager', 'workers', 'full']
-    if(args.type[0] not in choices):
-        print('Deployment/shutdown type not recognized or entered defaulted to the full option.\n')
-        args.type = 'full'
+    choices = ['manager', 'workers', 'full']
+    if (args.type[0] not in choices):
```

Comment:
hmm, why are we using List (`args.type[0]`) here?

Reply:
The positional statement comes in list format by default. Rather than try to change the format I just continue with it so not to add additional complexity.  TI will still need to address the case with multiple positional arguments.

Reply:
While fixing the other issues, I was able to fix this as well in a non-complex way.